### ISSUE TITLE
07/09/23_NYP_Android__PurpleRides_Tag for accessibility rides in my rides are not according to figma_1980

### ISSUE THREAD

**Issue by krupal-14** [nammayatri/nammayatri#3028]
#1980 
[Tag for accessibility rides in my rides are not according to figma](https://drive.google.com/file/d/1MDOep6-qZhRimzdIem568Y4i5t3uXN09/view?usp=sharing)
[Figma](https://www.figma.com/file/EXIHjZ0cvbAmYlzBcIbiDW/Accessibility-Updates-for-Namma-Yatri?type=design&node-id=3-805&mode=design&t=JUefVhq7l2X51EL6-0)

Steps to Reproduce
1.Launch the apk.
2.Login with valid credentials.
3.Initiate a ride.
4.Driver accepted the ride and completed.
5.Go to My rides and observe.

Expected result
Accessibility rides tag should be according to figma.

Actual result
Accessibility  rides tag is not displaying according to figma.

**Comment by krupal-14** [comment]
Still it is not according to figma.reopening


![Image](https://github.com/nammayatri/nammayatri/assets/132905061/739a8d02-7999-4f96-b217-87117901260d)

**Comment by MercyQueen** [comment]
Fixed.

**Comment by krupal-14** [comment]
Retested,working fine
